links.html

```diff
@@ -4,24 +4,29 @@
     <title>sanjay’s CSD120</title>
 </head>
 <body>
-        <header><nav></nav></header>
-        <main>
+         
+    <header>
         <nav>
-        <ul>
-        <li><a href="index.html">home</a> </li>
-        <li><a href="links.html">Links</a> </li>
-        <li><a href="glossary.html">Grossary</a></li>
-        </ul>
+            <ul>
+                <li><a href="index.html">Home</a> </li>
+                <li><a href="links.html">Links</a> </li>
+                <li><a href="glossary.html">Glossary</a></li>
+            </ul>
```

Comment:
The header navigation links should be in the `<header>` element as per lab instructions